.travis.yml

```diff
@@ -28,13 +40,13 @@ before_script:
 
   - composer validate
   - php bin/configure --default
-  - composer update --no-dev
+  - COMPOSER_MEMORY_LIMIT=-1 composer install --optimize-autoloader --no-interaction
   - npm install
-  - composer build
-  - php bin/console claroline:install --env=test -vvv
```

Comment:
When `--env=test`, it fails with an error about `services.yml`

@LaurentGruber, est-ce qu'on a besoin de cela?

Reply:
Could you provide the failure stack trace? Also, make sure to empty the `var/cache` directory before running it.

Reply:
It seems to work now with `export APP_ENV=test`, thanks